pom.xml

```diff
@@ -902,7 +902,7 @@
         <dependency>
             <groupId>com.graphql-java</groupId>
             <artifactId>graphql-java-extended-scalars</artifactId>
-            <version>${graphql.version}</version>
+            <version>20.0</version>
```

Comment:
Why don't we update this?

Reply:
I wonder if it makes sense to have the version variable then if it's only used 50% to define the version?

Reply:
That's a valid point particularly now that renovate manages the version. I will remove it.